src/app/templates/app/license_requests.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'app/base.html' %}
+{% load static %}
+{% load staticfiles %}
+{% block body1 %}
+
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.css"> 
+
+<div id="messages" class="messages">
+</div>
+<p class ="lead"> {{ error }}</p>
+<div class="panel panel-default">
+<div class="panel-heading"> <p class="lead">License Requests List</p> </div>
+<div class="panel-body">
+    <table id="requests_table" class="display dataTable">
+      <thead>
+        <tr>
+          <th>Fullname</th>
+          <th>Short Identifier</th>
+          <th>Submission Date</th>
+        </tr>
```

Comment:
This page shows just these 3 variables of the submitted license request. We need to create template/url to view more info about them. Add a link to that page here.

Reply:
Yes, actually that was a resolution we got in the call in which we discussed about the design of the database. The three variables showed in the list view are just for filtering, but when user clicks on any row, it will be redirected to a new template/url wich will contain the information of the license/request.
Would you suggest to add a click listener on the entire row? @rtgdk . Or one more column having for each row a clickable "View license" message?

Reply:
You can have a link to the template/url for the whole row. Like in HTML
`<a href="/.../{particular_license_page}"><tr>.....</tr></a>`